lib/perl/Genome/FeatureList.pm

```diff
@@ -421,11 +421,17 @@ sub generate_converted_bed_file {
     }
     my $original_md5 = Genome::Sys->md5sum($original_file_path);
 
+    my $result_users = delete($args{result_users});
+    $result_users->{'feature-list'} = $self;
+    $result_users->{requestor}    ||= $reference;
+    $result_users->{sponsor}      ||= Genome::Sys->current_user;
```

Comment:
Or maybe make dummy values that would allow these to be separated from cases where the sponsor is a person, etc.?